umodbus/serial.py

```diff
@@ -292,7 +292,7 @@ def _send(self, modbus_pdu: bytes, slave_addr: int) -> None:
     def _send_receive(self,
                       modbus_pdu: bytes,
                       slave_addr: int,
-                      count: bool) -> bytes:
+                      count: bool) -> bytes|None:
```

Comment:
```suggestion
                      count: bool) -> Union[bytes, None]:
```

I think we need to take this oldschool typing drawback to be compatible with older MicroPython versions. Plus, `flake8` was unhappy due to the missing spaces around `|`